core/src/tile/tileWorker.cpp

```diff
@@ -144,6 +185,10 @@ void TileWorker::setScene(std::shared_ptr<Scene>& _scene) {
     }
 }
 
+void TileWorker::notifyAll() {
+    m_condition.notify_all();
+}
+
```

Comment:
As long as !rastersReady in TileWorker the task is re-enqueued, so it should loop until all rasters are available. 
Commenting out line 189 seems to make no difference here - maybe this was needed earlier to work around a different problem?